numpy/core/tests/test_multiarray.py

```diff
@@ -342,6 +342,10 @@ def test_array_cont(self):
         assert_(np.ascontiguousarray(d).flags.c_contiguous)
         assert_(np.asfortranarray(d).flags.f_contiguous)
 
+    def test_npy_char_deprecation(self):
```

Comment:
I can't get it to work the test_deprecation way.

```
class TestNPY_CHAR(_DeprecationTestCase):
    def test_npy_char_deprecation(self):
        from numpy.core.multiarray_tests import npy_char_deprecation
        self.assert_deprecated(npy_char_deprecation)
```

does not work, it says `AssertionError: No error raised during function call`. How is the test supposed to look like?